.gitignore

```diff
@@ -0,0 +1 @@
+.idea/

```

Comment:
add `node_modules` and `lib` like [here](https://github.com/DevExpress/testcafe/blob/master/.gitignore)